app/commands.py

```diff
@@ -2311,6 +2311,9 @@ async def clan_create(ctx: Context) -> str | None:
     if await clans_repo.fetch_one(tag=tag):
         return "That tag has already been claimed by another clan."
 
+    if await clans_repo.fetch_one(tag=tag) == tag.upper():
```

Comment:
This check adds.. nothing? The same check already exists above and should be used. If that check for whatever reason does not work, it should be fixed.